yt_dlp/extractor/tv4.py

```diff
@@ -20,19 +23,25 @@ class TV4IE(InfoExtractor):
                 sport/|
             )
         )(?P<id>[0-9]+)'''
-    _GEO_COUNTRIES = ['SE']
+    _GEO_BYPASS = False
```

Comment:
Why?

Reply:
per dirkf's comment here https://github.com/yt-dlp/yt-dlp/pull/5649#discussion_r1039935656 that xff is futile

Reply:
Then why r we passing go verification headers? Am I misunderstanding something?

Reply:
`_GEO_BYPASS` is for `-xff`, which has no effect
`geo_verification_headers()` are for `--geo-verification-proxy`, which does work